demo/meeting/src/containers/VideoStreamMetrics/index.tsx

```diff
@@ -0,0 +1,205 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import {
+  useMediaStreamMetrics,
+  useAudioVideo,
+  PopOverHeader
+} from 'amazon-chime-sdk-component-library-react';
+
+import MediaStatsList from '../../components/MediaStatsList/index';
+import MetricItem from '../../components/MediaStatsList/MetricItem';
+import { StyledMediaMetricsWrapper } from '../../components/MediaStatsList/Styled';
+
+function isValidMetric(metric: any) {
+  return typeof metric === 'number' && !Number.isNaN(metric);
+}
+
+interface Props {
+  /** The Chime attendee ID */
+  attendeeId: string;
+}
+
+export const VideoStreamMetrics: React.FC<Props> = ({ attendeeId }) => {
+  const audioVideo = useAudioVideo();
+  const { videoStreamMetrics } = useMediaStreamMetrics();
+  const streamMetric = videoStreamMetrics[attendeeId];
+  const ssrcArray = streamMetric ? Object.keys(streamMetric) : [];
```

Comment:
So do we also display video metrics for local attendee here? If so, is it duplicate with the local metrics?

Reply:
What do you mean? The local video metrics is duplicate with the local metrics?

Reply:
Oh, so just to make it consistent, I think it would be good to show metrics for local attendee here in roster. I don't think it is harmful to have it. What do you think?